tests/unit/introspection.tcl

```diff
@@ -503,13 +503,35 @@ start_server {tags {"introspection"}} {
         assert_match {*'replicaof "--127.0.0.1"'*wrong number of arguments*} $err
     } {} {external:skip}
 
+    test {redis-server command line arguments - allow passing option name and option value in the same arg} {
+        start_server {config "default.conf" args {"--maxmemory 700mb" "--maxmemory-policy volatile-lru"}} {
+            assert_match [r config get maxmemory] {maxmemory 734003200}
+            assert_match [r config get maxmemory-policy] {maxmemory-policy volatile-lru}
+        }
+    } {} {external:skip}
+
+    test {redis-server command line arguments - wrong usage} {
```

Comment:
bad title for the test?
wrong usage that we support anyway (test verifies it works).

Reply:
yes. bad title. i want to add it in case we break it somehow in the future.. (and i did not think of a good title ...)